Bellerophon/Bellerophon/Sources/BPManager.swift

```diff
@@ -105,31 +105,34 @@ public class BellerophonManager: NSObject {
         }
     }
 
-    private func stopTimer() {
+    internal func stopTimer() {
         retryTimer?.invalidate()
         retryTimer = nil
     }
 
-    private func performForceUpdate() {
+    internal func performForceUpdate() {
         delegate?.shouldForceUpdate()
     }
 
-    private func displayKillSwitch() {
+    internal func displayKillSwitch() {
         if !killSwitchWindow.keyWindow {
             killSwitchView.frame = killSwitchWindow.bounds
             delegate?.bellerophonWillEngage?(self)
-            killSwitchWindow.makeKeyAndVisible()
+            // if there's no such keyWindow in the application, do not show killSwitchWindow
```

Comment:
It might have the chance that there is no key window in the application using this pod, e.g. a new framework that uses Bellerophon as a dependency, or unit testing targets for Bellerophon. In such cases, at running time, there would be a fatal error of "NSInternalInconsistencyException", "props must have a valid clientID". So this is used to avoid that. 

Thanks for the advice to subclass BPManager and override methods that related to UI part to do the trick :) Will do that!